### PR TITLE
GPII-3619: Fix some fixtures that cause the failure at running tests/MultiSettingsHandlerTests.js

### PR DESCRIPTION
**Pull request by cindyli**

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
Thank you so much, @cindyli.  I'm reviewing to try to understand why that change was necessary and see if we can simplify the new setup.  The previous tests used a member variable instead:

https://github.com/GPII/universal/blob/master/gpii/node_modules/testing/src/Testing.js#L382

My assumption is that the member variable holds the resolved version of whatever's found in `options.settingsHandlers`, and should have been resolved and created here:

https://github.com/GPII/universal/blob/master/gpii/node_modules/testing/src/Testing.js#L337

I'd like to understand why we can use options directly now for this check when we didn't need to previously.  I want to make sure that there isn't some kind of remaining problem that will break other things that expect to be able to inspect the resolved `{test}.settingsHandlers` member variable, as there are many others.  If it turns out that we can use `{test}.options.settingsHandlers` directly everywhere, my next question is, can we eliminate the resolution and expansion?  I'll try out more of the broken tests and my branch to see what I can discover.

File: gpii/node_modules/testing/src/Fixtures.js [hunk 1]
```diff
@@ -308,13 +308,13 @@ fluid.defaults("gpii.test.checkSequence", {
...
     gradeNames: ["gpii.test.couchSequenceGrade"],
     sequenceElements: {
-        login: {
-            gradeNames: "gpii.test.loginSequence",
-            priority: "after:startServer"
-        },
         snapshot: {
             gradeNames: "gpii.test.snapshotSequence",
-            priority: "after:login"
+            priority: "after:startServer"
```
Comment (the-t-in-rtf) [inline gpii/node_modules/testing/src/Fixtures.js:313]: This seems a lot more likely to be the real solution to the problem.

Comment (the-t-in-rtf) [reply gpii/node_modules/testing/src/Fixtures.js:313]: Confirmed, simply changing the order fixes the test failure in my branch.

Comment (the-t-in-rtf) [reply gpii/node_modules/testing/src/Fixtures.js:313]: Looks like we picked up 1-2 other tests with just this fix.  Thanks again, @cindyli.

**Comment by the-t-in-rtf** [comment]
Thanks again, @cindyli, it was very helpful.

**Comment by cindyli** [comment]
You are welcome. Great it helps.

**Pull request closed without merging**